core/src/v2/schema.rs

```diff
@@ -185,6 +185,8 @@ impl_type_simple!(isize, DataType::Integer, DataTypeFormat::Int64);
 impl_type_simple!(u64, DataType::Integer, DataTypeFormat::Int64);
 impl_type_simple!(u128, DataType::Integer, DataTypeFormat::Int64);
 impl_type_simple!(usize, DataType::Integer, DataTypeFormat::Int64);
+impl_type_simple!(serde_json::Value, DataType::String);
+impl_type_simple!(serde_yaml::Value, DataType::String);
```

Comment:
This does not conform to actix_web Responder implementation as Value does not implement String Responder. 

You would usually wrap Value as `web::Json<Value>` and it will output object. Same when Value is used as part of a Serializable struct - it will use wrapper's implementation.

If you need to output Value or your strust as a String you would usually need to implement your JsonAsString wrapper, where you can define this mapping and implement custom Apiv2Schema.

Reply:
Thanks. Got it.